client/consul/sync_test.go

```diff
@@ -47,13 +50,15 @@ func TestConsulServiceRegisterServices(t *testing.T) {
 		return
 	}
 	task := mockTask()
-	if err := cs.SyncTask(task); err != nil {
+	cs.SetServiceIdentifier(fmt.Sprintf("%s-%s", allocID, task.Name))
```

Comment:
might be worth making `fmt.Sprintf("%s-%s", allocID, task.Name)` a helper function in the consul package